templates/user/dashboard/navbar.tmpl

```diff
@@ -11,7 +11,7 @@
 					<div class="ui header">
 						{{.i18n.Tr "home.switch_dashboard_context"}}
 					</div>
-					<div class="items">
+					<div class="scrolling menu" style="border-radius: 0px !important;box-shadow: none !important;border-bottom: 1px solid rgba(34, 36, 38, 0.15);">
```

Comment:
Inline style should be moved to less/CSS class

Reply:
Sorry I missed that!

Reply:
@zeripath no need to be sorry. This is why there is a policy to have at least two acceptance as sometimes one person may miss something that another doesn't (and vice versa).

Reply:
done

Reply:
@lafriks can you check please sir